src/test/resources/TCS.feature

```diff
@@ -3,11 +3,13 @@ Feature: Navigation bar
     Without logging in
     I can the login workflow
 
+    @test
     Scenario: I can try to login using a correct and wrong password.
         Given I navigate to www.saucedemo.com
         When Login using a correct password
         Then Login using incorrect password
```

Comment:
Y en el Then finalmente validamos el comportamiento descrito en el when, "Then the user should be able to access to the inventory page"